client/state/imports/actions.js

```diff
@@ -55,38 +54,39 @@ const expiryOrder = ( siteId, importerId ) =>
 const importOrder = importerStatus =>
 	toApi( Object.assign( {}, importerStatus, { importerState: appStates.IMPORTING } ) );
 
-const apiStart = () => Dispatcher.handleViewAction( { type: IMPORTS_FETCH } );
-const apiSuccess = data => {
-	Dispatcher.handleViewAction( { type: IMPORTS_FETCH_COMPLETED } );
+const apiStart = () => ( { type: IMPORTS_FETCH } );
+
+const apiSuccess = dispatch => data => {
+	dispatch( { type: IMPORTS_FETCH_COMPLETED } );
```

Comment:
`apiSuccess`, `apiFailure` and `receiveImporterStatus` should be classic action creators: function that returns an action object (with a `type` field etc.).